src/js/components/covid19/amountsVisualization/AmountsVisualization.jsx

```diff
@@ -26,7 +25,7 @@ import {
 
 const propTypes = {
     overviewData: PropTypes.object,
-    width: PropTypes.number
+    width: PropTypes.number,
```

Comment:
I'm guessing this was added by your editor but it might be creating a lint warning